### ISSUE TITLE
UselessDocComment false positive

### ISSUE THREAD

**Issue by simPod** [slevomat/coding-standard#707]
Hi, following code reports

> Function a() does not need documentation comment

```php
<?php

declare(strict_types=1);

use Ds\Map;

/**
 * @return Map<stdClass,int>
 */
function a() : Map
{
    return new Map();
}
```

I assume it's bug in the sniff (?)

**Comment by kukulich** [comment]
@simPod What version of Slevomat CS do you use?

**Comment by simPod** [comment]
Sorry for not specifying :/ `5.0.4`. It's fixed in the new one, right? :D

**Comment by kukulich** [comment]
@simPod It should be fixed in master, can you test please?

**Comment by simPod** [comment]
Confirmed b5a3baccdbc2607bd4f30064adcf9d4854779bab doesn't report it anymore. Any chance for patch release?

**Comment by kukulich** [comment]
Duplicate of #673

**Comment by kukulich** [comment]
I hope I will release https://github.com/slevomat/coding-standard/milestone/5 soon...

**Comment by simPod** [comment]
Thanks a lot
